mungegithub/mungers/blunderbuss.go

```diff
@@ -112,7 +112,7 @@ func (b *BlunderbussMunger) Munge(obj *github.MungeObject) {
 		// makes three buckets, we shouldn't have many 10k+
 		// line changes.
 		fileWeight = int64(math.Log10(float64(fileWeight))) + 1
-		fileOwners := b.features.Repos.LeafAssignees(*file.Filename)
+		fileOwners := b.features.Repos.LeafReviewers(*file.Filename)
```

Comment:
I'm going to change this line and get this code merged. This is the only thing that is unsafe to merge.